UPGRADE.md

```diff
@@ -74,6 +74,17 @@ If you need to access a route parameter, use react-router's `useParams` hook ins
 };
 ```
 
+We used to inject a `syncWithLocation` prop set to `true` to the `List` components used in a `Resource`. As we removed the props injection, we also inverted this prop and renamed it `disableSyncWithLocation`. This means you now have to explicitly opt out the synchronization of the list parameters with the browser location:
```

Comment:
This isn't very clear. I'm putting myself in the shows of someone who has never seen this prop and I wonder if I need to add it or not...

Reply:
Better now?